tests/enforcer/gce_firewall_enforcer_test.py

```diff
@@ -18,37 +18,38 @@
 import copy
 import json
 import threading
-import mock
 import unittest
+import mock
+
+from parameterized import parameterized
 
 from tests.enforcer import testing_constants as constants
 from tests.unittest_utils import ForsetiTestCase
 
 from google.cloud.security.enforcer import gce_firewall_enforcer as fe
 
-
 class HelperFunctionTest(ForsetiTestCase):
     """Unit tests for helper functions."""
 
     def test_get_network_name_from_url(self):
         """Verify that we can get the network name given a network url."""
-        url = ('https://www.googleapis.com/compute/v1/projects/%s/global/'
-               'networks/%s' % (constants.TEST_PROJECT, constants.TEST_NETWORK))
-        self.assertEquals(constants.TEST_NETWORK,
-                          fe.get_network_name_from_url(url))
+        url = ('https://www.googleapis.com/compute/{}/projects/'
+               'example.com:testing/global/networks/'
+               'expected-network').format(fe.API_VERSION)
+        self.assertEqual('expected-network',
+                         fe.get_network_name_from_url(url))
 
     def test_build_network_url(self):
         """Verify that we can get a url from project and network name."""
-        expected_url = ('https://www.googleapis.com/compute/v1/projects/'
-                        '%s/global/networks/%s' %
-                        (constants.TEST_PROJECT, constants.TEST_NETWORK))
-
-        self.assertEqual(expected_url,
-                         fe.build_network_url(constants.TEST_PROJECT,
-                                              constants.TEST_NETWORK))
+        self.assertEqual(
+            fe.build_network_url(
```

Comment:
By convention we put the expected value first for assertEqual and the actual value second. So fe.build_network_url(..) should be the second argument instead of the first.